content/05.use-cases-c.md

```diff
@@ -37,6 +37,21 @@ In (Figure @fig:github-diagram), we highlight several elements of good repositor
 GitHub also allows users to describe and store information about software and code dependencies, ensuring that code can run using the same version of software as when it was initially developed.
 This can be achieved through the access of GitHub repositories containing the necessary releases of such software, or through the use of automated Github Actions with specific software versions.
 
+### Automation
+
+<!--*Contributors to this section: PHPB -->
+
+Automation has a strong potential to expand the scale and pace of research in ecology and evolution [see @doi:10.1126/science.abi4692].
+Automation frameworks can be applied from the collection, analysis, scrutiny, to the integration and deployment of data and code in EEB research.
+In this context, small modifications to code and data can be frequently committed and automatically tested (as in continuous integration and continuous deployment practices), allowing for the early detection and correction of errors [@doi:10.1109/MS.2014.58], ultimately mitigating the erosion of confidence in scientific development due to software errors [see @doi:10.12688/f1000research.5930.2].
+In addition to increasing scientific rigor and confidence in ecological software [@doi:10.1890/080141], can help advance in the public dissemination and quality of ecological data [@doi:10.1073/pnas.1710231115].
+The adoption of integrative automation workflows have been highly encourage in areas of EEB, including in predictive ecology [@doi:10.1111/ele.13994], and have been applied to long-term ecological studies [_e.g._, @doi:10.1371/journal.pbio.3000125] and in the management of species at risk information [_e.g._, @doi:10.1038/s41597-022-01381-8].
+
+Automation can be integrated into GitHub repositories through the GitHub Actions feature [@url:https://github.com/features/actions], or through alternative automation systems (_e.g._, Circle CI[@url:https://circleci.com], Travis CI[@url:https://www.travis-ci.com]).
+Users can set up workflows associated with their repositories that are triggered by events (_e.g._, push, pull request, or at specified times) for remote servers (runners) to perform user-specified steps and actions.
+These actions are highly configurable and have numerous applications, such as automatically running analyses and creating figures when data or code are updated, incorporating changes to websites or applications, testing modifications to software (_e.g._, R or Python libraries).
```

Comment:
this could maybe be shortened as several examples have already been given.

Reply:
OK that works for me. thanks!